src/components/security_manager/src/crypto_manager_impl.cc

```diff
@@ -261,6 +261,8 @@ bool CryptoManagerImpl::Init() {
 #endif
   }
 
+  SSL_CTX_set_security_level(context_, get_settings().security_level());
```

Comment:
@VadymLuchko I suppose this function is not present in 1.0.2 so most likely you need to wrap this up with ifdef for OPENSSL1_1_VERSION

Reply:
fixed in be9c3c1340e374d266f87fb35a5048dfe803a4b2